spring-cloud-sleuth-brave/src/main/java/org/springframework/cloud/sleuth/brave/sampler/SamplerConfiguration.java

```diff
@@ -40,7 +40,7 @@
 // This is not auto-configuration, but it was in the past. Leaving the name as
 // SamplerAutoConfiguration because those not using Zipkin formerly had to
 // import this directly. A less precise name is better than rev-locking code.
-public class SamplerAutoConfiguration {
+public class SamplerConfiguration {
```

Comment:
Right now the class header mentions some legacy stuff without any sort of deprecation warning. I would deprecate this and move all the code to `autoconfigure` with an `@Import` to the new class so that those who are using the deprecated  class are not affected.